owncloudApp/src/main/java/com/owncloud/android/ui/activity/UploadListActivity.java

```diff
@@ -74,7 +75,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setFile(null);
 
         // setup toolbar
-        setupRootToolbar(getString(R.string.uploads_view_title), false);
+        setupRootToolbar(getString(R.string.uploads_view_title), false, true);
```

Comment:
Do we really need to fetch the avatar in this activity as well? Don't we have it in cache or something like that? Just asking

Reply:
If we navigate to Uploads view the avatar is not going to be requested. We get the capabilities and there is a condition `shouldFetchAvatar` to fetch the avatar from the server.

Reply:
In any case, this is not needed here